net/crowdsec-firewall-bouncer/files/crowdsec-firewall-bouncer.initd

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh /etc/rc.common
+# (C) 2021 Gerald Kerma
+
+START=99
+USE_PROCD=1
+NAME=crowdsec-firewall-bouncer
+PROG=/usr/bin/cs-firewall-bouncer
+CONFIG=/etc/crowdsec/bouncers/crowdsec-firewall-bouncer.yaml
+BACKEND=iptables
+
+service_triggers() {
+        procd_add_reload_trigger crowdsec-firewall-bouncer
+}
+
+boot()
+{
+	rc_procd start_service
+}
+
+init_config() {
+	config_load crowdsec
+	config_get data_dir crowdsec backend "${BACKEND}"
+	
+        sed -i "s,^\(\s*mode\s*:\s*\).*\$,\1$BACKEND," $CONFIG
+}
+
+start_service() {
+	init_config
+	
+	procd_open_instance
+	procd_set_param command "$PROG" -c "$CONFIG"
+	procd_close_instance
+}
+
+reload_service() {
+	procd_send_signal $PROG
+}
+
+stop_service()
+{
+	rc_procd killall "$PROG" 2>/dev/null
+}
```

Comment:
Are you sure the default `stop()` function doesn't terminate the process correctly?

Reply:
removed
thanks